db/seeds.rb

```diff
@@ -5,3 +5,24 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+categories = Category.create([{title: "Apparel"},{title: "Snowboards"}])
+
+apparel = Category.first.items.create([{name: "Hoody",
+                      description: "Keeps you slightly warm",
+                      price: 80},
+
+                     {name: "Sweet Jacket",
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Space inside { missing.